pom.xml

```diff
@@ -499,7 +505,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <raml-module-builder.version>33.2.4</raml-module-builder.version>
+    <raml-module-builder.version>34.0.0</raml-module-builder.version>
+    <okapi-testing.version>4.14.1</okapi-testing.version>
```

Comment:
Personally, I don't think there is value in extracting an intermediary property for versions that are only used by a single artefact.